scanpy_scripts/lib/_norm.py

```diff
@@ -12,6 +13,13 @@ def normalize(adata, log_transform=True, **kwargs):
     """
     sc.pp.normalize_total(adata, **kwargs)
     if log_transform:
-        sc.pp.log1p(adata)
+        # Natural logarithm is the default by scanpy, if base is not set
+        base = math.e
```

Comment:
I don't understand this. If Natural (neperian) is the default in Scanpy, why are you defining it?

Is this related to the bump version in Scanpy?

Reply:
it is mostly to explicitly set it in adata.uns. There seems to be an issue by which if you call sc.pp.log1p without the base explicitly given, it doesn't record the base on the uns, and as a result later down the line you have no idea on which base the log was called.